llvm/lib/Target/X86/X86.td

```diff
@@ -1429,7 +1444,8 @@ def ProcessorFeatures {
                                          TuningFastScalarShiftMasks,
                                          TuningBranchFusion,
                                          TuningSBBDepBreaking,
-                                         TuningInsertVZEROUPPER];
+                                         TuningInsertVZEROUPPER,
+                                         TuningAvoidMFENCE];
```

Comment:
Please can you add TuningAvoidMFENCE to BtVer2Tuning as well?

Reply:
Done!